docker/configure_workers_and_start.py

```diff
@@ -512,9 +512,10 @@ def generate_worker_files(environ, config_path: str, data_dir: str):
     )
 
     # Supervisord config
+    os.makedirs('/etc/supervisor', exist_ok=True)
     convert(
         "/conf/supervisord.conf.j2",
-        "/etc/supervisor/conf.d/supervisord.conf",
+        "/etc/supervisor/supervisord.conf",
```

Comment:
Previously, we would `apt-get install supervisor`, which would install a config file in `/etc/supervisor/supervisord.conf`, which would include `/etc/supervisor/conf.d/*.conf`. We now install the generated config file directly to `/etc/supervisor/supervisord.conf` instead.

Reply:
This will make it trickier to add a `/etc/supervisor/conf.d/postgres.conf` file from a `COPY` step in the Dockerfile; I'm not sure it's an improvement. (I could be convinced.)

I suppose the include could be added back in to our config when it becomes useful.

Reply:
yeah, if we've got a usecase for the include, we can just add it back into `supervisord.conf.j2`.

Reply:
in fact, lemme do it now.

Reply:
done in [dadfa72](https://github.com/matrix-org/synapse/pull/12464/commits/dadfa72dd04767a3da33ba6de797523191d868df)